CHANGELOG.md

```diff
@@ -268,6 +268,7 @@ the stability guarantees we provide for pre-releases.
 
 ### IMPROVEMENTS
 
+- `[crypto/merkle]` innerHash is much more efficient ([#1921](https://github.com/cometbft/cometbft/pull/1921))
```

Comment:
this must be in a separate file as per https://github.com/cometbft/cometbft/blob/main/CONTRIBUTING.md#changelog

Reply:
Oh, damn I totally missed that, I've made the new file and moved it. Thank you @melekes!